src/Request/Admin/RequestHomebyAdmin.js

```diff
@@ -47,27 +47,35 @@ class RequestHomebyAdmin extends React.Component {
                 <div className="col-md-8" style={{'margin-left':'80px'}}>
                     <h2 className="PageTitle">View All Request</h2><br/>
                     <div className="contents">
-                        <Form inline>
+                    <Form inline>
                             <Form.Control as="select">
                                 <option value="30">Last 30 Days</option>
                                 <option value="60">Last 60 Days</option>
                                 <option value="90">Last 90 Days</option>
                                 <option value="120">Last 120 Days</option>
                             </Form.Control>
-                            <Form.Control type='date' style={{'margin-left':'30px'}}/>
+                            <Form.Control type='date' style={{'margin-left':'30px','margin-right':'15px'}}/>~<Form.Control type='date' style={{'margin-left':'15px'}}/>
                             <Form.Control as="select" style={{'margin-left':'30px'}}>
-                                <option value="title">Title</option>
-                                <option value="content">Category</option>
-                                <option value="author">Service</option>
+                                <option value="title">Customer ID</option>
+                                <option value="content">Customer Name</option>
```

Comment:
Since you have customer ID and customer name in the quick search dropdown, do we need to display the customer id/name in the request list table in admin?

![image](https://user-images.githubusercontent.com/50813726/106837805-1e3f5b80-6669-11eb-8698-f460cba7ec3e.png)

Reply:
I followed the mock-up, but if it does not need, I delete it

Reply:
thank you!